operators/src/pro/meta/wrapper.rs

```diff
@@ -154,17 +173,23 @@ where
         let span = (self.span)();
         let _enter = span.enter();
 
-        tracing::trace!(event = "query");
+        tracing::trace!(event = "query", path = ?self.path);
```

Comment:
isn't this redundant with line 157?

Reply:
https://github.com/geo-engine/geoengine/pull/780/commits/f088e93e3762d8e0ca51deafcffd71336be94aad